uber/models/__init__.py

```diff
@@ -676,10 +676,48 @@ def attendees_share_departments(self, first, second):
 
         def admin_can_create_attendee(self, attendee):
             admin = self.current_admin_account()
+            if admin.full_registration_admin:
+                return True
+            
             if attendee.badge_type == c.STAFF_BADGE:
                 return admin.full_shifts_admin
```

Comment:
Sanity check: should we be immediately returning in cases like this?  Like, let's say that someone has a ``c.STAFF_BADGE`` and doesn't have ``full_shifts_admin`` access, but they would qualify for one of the ``if`` statements below.  Currently we'd return ``False`` even if a later condition would result in ``True``.

I wonder if we should instead be only returning immediately if the result is ``True`` and otherwise passing to the next if-check.

Reply:
That is an excellent point! I will implement that.

Reply:
... after implementing that, and while testing it, I realized that this would, for example, let admins create Staff badges for people at-will as long as they also gave them a panelist ribbon. So. I'm going to un-implement that.

If this check is failed, all that happens is that the badge's status is set to Pending and it therefore gets put in the badge approval queue, so it's okay for it to be strict.